manifests/runtimes.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: node.k8s.io/v1
```

Comment:
Add `runtimes` to the list at https://github.com/k3s-io/k3s/blob/96ebb96317fba98e6913f532ed9f715aa4365a83/pkg/cli/cmds/stage.go#L9